src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -1479,54 +1414,6 @@ namespace HVACUnitarySystem {
 			}
 		}
 
-		// Find the number of zones (zone Inlet Nodes) attached to an air loop from the air loop number
```

Comment:
Moved this code to sizing since I needed to know the control zone fraction in order to autosize the min/max temperature limits.